test/functional/eval/fnamemodify_spec.lua

```diff
@@ -3,9 +3,19 @@ local clear = helpers.clear
 local eq = helpers.eq
 local iswin = helpers.iswin
 local fnamemodify = helpers.funcs.fnamemodify
+local getcwd = helpers.funcs.getcwd
 local command = helpers.command
 local write_file = helpers.write_file
 
+-- for :p we must be prepared to handle Windows style paths
+function eq_slashconvert(expected, got)
+  if iswin() then
+    got = got:gsub([[/]], [[\]])
+  end
+
+  eq(expected, got)
```

Comment:
could use `helpers.alter_slashes()`.``

```suggestion
  local write_file = helpers.alter_slashes
  eq(alter_slashes(expected), (got))
```